contentcuration/contentcuration/tests/views/test_views_internal.py

```diff
@@ -784,3 +788,362 @@ def test_associates_extra_fields_with_root_node(self):
             self.fail("Channel was not created")
 
         self.assertEqual(c.chef_tree.extra_fields["modality"], "CUSTOM_NAVIGATION")
+
+
+class ApiAddRemoteNodesToTreeTestCase(StudioTestCase):
+    """
+    Tests for contentcuration.views.internal.api_add_nodes_to_tree function.
+    For adding nodes that already exist on Studio to a cheffed tree.
+    """
+
+    def _make_node_data(self):
+        random_data = mixer.blend(SampleContentNodeDataSchema)
```

Comment:
Nice! I haven't seen this before

Reply:
Hah, I copied this from elsewhere in the same file - so I can take 0 credit :)